rollup.config.js

```diff
@@ -0,0 +1,11 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
+export default {
+  input: 'main.js',
+  output: {
+    dir: 'dist',
+    format: 'cjs',
+    sourcemap: true
+  },
+  plugins: [nodeResolve({ resolveOnly: ["base58-universal"] })]
```

Comment:
nitpick: would be good to have put some comment about the why we add `"base58-universal"` here :)

Reply:
Yup done :) let me know if its okay